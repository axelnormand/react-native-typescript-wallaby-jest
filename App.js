import React from 'react';
import App from './src/app/App';

/**
 * Create React Native App expects App.js to be in <root> folder
 * So just wrap our App.js in src/ here
 */
export default App;
