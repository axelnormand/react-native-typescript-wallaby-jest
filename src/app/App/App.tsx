import React from 'react';
import { Text } from 'react-native';

const App = () => (
  <Text>Hello World!</Text>
);

export default App;
