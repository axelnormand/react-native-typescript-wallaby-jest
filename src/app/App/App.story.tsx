import { storiesOf } from '@storybook/react-native';
import React from 'react';
import App from './App';

storiesOf('App', module).add('normal render', () => <App />);
