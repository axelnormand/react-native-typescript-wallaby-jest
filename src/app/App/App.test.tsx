import { shallow } from 'enzyme';
import React from 'react';
import App from './App';

test('App renders', () => {
  const wrapper = shallow(<App />);
  expect(wrapper).toMatchSnapshot();
});
