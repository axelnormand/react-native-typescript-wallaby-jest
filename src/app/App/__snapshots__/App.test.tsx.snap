// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
>
  Hello World!
</Text>
`;
